_posts/2020-12-22-ols.md

```diff
@@ -0,0 +1,76 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Open Life Science program & the Galaxy community: implication in OLS-2 and invitation to apply to the next cohort'
+tags: training
+supporters:
+- elixir
+- eosc
+- denbi
+---
+
+[Open Life Science (OLS)](https://openlifesci.org/) is an **online mentoring & training program** to help **individuals and stakeholders in research** to become **Open Science ambassadors**. Participants of this program learn essential knowledge required to create, lead, and sustain an Open Science project, connect with members across different communities, backgrounds, and identities by sharing their experiences and expertise.
+
+The first two cohorts have been run in 2020, in which the Galaxy community was successfully involved. Applications are now open for the 3rd cohort (OLS-3) which will take place from **February to May 2021**. This cohort is supported by EOSC-Life. We would love to see a strong representation of the Galaxy community there.
+
+# 2 Galaxy-related projects in OLS-2
+
+In the 2nd cohort (OLS-2), [33 diverse projects](https://openlifesci.org/ols-2/projects-participants/#projects) were leaded by [51 participants](https://openlifesci.org/ols-2/projects-participants/#participants), mentored by [35 mentors](https://openlifesci.org/ols-2#mentors) and supported by [65 experts](https://openlifesci.org/ols-2#experts). And among them, 2 projects were Galaxy-related and several participants, mentors and experts were from the Galaxy community ([Beatriz Serrano-Solano](https://openlifesci.org/ols-2/projects-participants#beatrizserrano), [Bérénice Batut](https://openlifesci.org/ols-2#bebatut), [Björn Grüning](https://openlifesci.org/ols-2#bgruening ), [Dave Clements](https://openlifesci.org/ols-2#tnabtaf), [Delphine Lariviere](https://openlifesci.org/ols-2#Delphine-L), [Hans-Rudolf Hotz](https://openlifesci.org/ols-2#hrhotz), [Mallory Freeberg](https://openlifesci.org/ols-2#malloryfreeberg), [Martin Cech](https://openlifesci.org/ols-2#martenson), [Maria Doyle](https://openlifesci.org/ols-2#mblue9), [Peter van Heusden](https://openlifesci.org/ols-2/projects-participants#pvanheus), [Tomas Klingström](https://openlifesci.org/ols-2#TKlingstrom), [Yo Yehudi](https://openlifesci.org/ols-2#yochannah), [Yvan Le Bras](https://openlifesci.org/ols-2#yvanlebras)).
```

Comment:
leaded -> led
And among them -> Among them

I am all for claiming Yo as part of community, but might want to check with her.